wisp/wisp.php

```diff
@@ -170,6 +170,22 @@ function wisp_ConfigOptions() {
             "Type" => "text",
             "Size" => 25,
         ],
+        "additional_ports" => [
```

Comment:
I think adding these in the middle of the array, in theory, would break how the wisp_GetOption function works (as it relies on the index as a fallback) and thus affect backward compatibility between all existing users - though I'll probably look into improving this myself during the next week if I have time.

Reply:
It looks like you're right - After moving them to the bottom of the array and testing, the previous settings I had stayed put in whatever field was now in that index, borking my settings.
I've moved these to the bottom of the array to maintain backwards compatibility 👍